src/components/common/FeatureBox/FeatureBox.module.scss

```diff
@@ -5,6 +5,27 @@
   text-align: center;
   margin-top: 40px;
 
+  .paragraph {
+    transition: 1s;
+  }
+
+  a:hover {
+    font-size: 25px;
```

Comment:
To nie jest najlepszy pomysł, cała strona się w tedy przesuwa w dół i skacze.

Fajnie, jakby się dało najechać na cały bloczek a nie tylko na kółko lub tekst